scripts/analyze_github_actions_usage.py

```
@@ -0,0 +1,390 @@
+PRICING_MACOS = {
+    "macos-latest": 0.08,  # $0.08 per minute
+    "macos-13": 0.08,
+    "macos-12": 0.08,
+    "macos-11": 0.08,
+}
+
+
+class GitHubActionsAnalyzer:
+    """Analyze GitHub Actions usage and costs."""
+
+    def __init__(self, token: str, owner: str, repo: str):
```

Ruff (0.14.13)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)